objax/io/checkpoint.py

```diff
@@ -89,6 +89,8 @@ def restore(self, vc: VarCollection, idx: Optional[int] = None):
             idx: index of the restored checkpoint.
             ckpt: full path to the restored checkpoint.
         """
+        assert isinstance(vc, VarCollection), \
+            f"Must pass a VarCollection to save; received type {type(vc)}."
```

Comment:
Nit: coding style consistency, also I believe you meant restore not save here.
```suggestion
            f'Must pass a VarCollection to restore; received type {type(vc)}.'
```